{
    "active_lock_reason": null,
    "assignee": {
        "avatar_url": "https://avatars.githubusercontent.com/u/15736910?v=4",
        "events_url": "https://api.github.com/users/zheng-xq/events{/privacy}",
        "followers_url": "https://api.github.com/users/zheng-xq/followers",
        "following_url": "https://api.github.com/users/zheng-xq/following{/other_user}",
        "gists_url": "https://api.github.com/users/zheng-xq/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/zheng-xq",
        "id": 15736910,
        "login": "zheng-xq",
        "node_id": "MDQ6VXNlcjE1NzM2OTEw",
        "organizations_url": "https://api.github.com/users/zheng-xq/orgs",
        "received_events_url": "https://api.github.com/users/zheng-xq/received_events",
        "repos_url": "https://api.github.com/users/zheng-xq/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/zheng-xq/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/zheng-xq/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/zheng-xq"
    },
    "assignees": [
        {
            "avatar_url": "https://avatars.githubusercontent.com/u/15736910?v=4",
            "events_url": "https://api.github.com/users/zheng-xq/events{/privacy}",
            "followers_url": "https://api.github.com/users/zheng-xq/followers",
            "following_url": "https://api.github.com/users/zheng-xq/following{/other_user}",
            "gists_url": "https://api.github.com/users/zheng-xq/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/zheng-xq",
            "id": 15736910,
            "login": "zheng-xq",
            "node_id": "MDQ6VXNlcjE1NzM2OTEw",
            "organizations_url": "https://api.github.com/users/zheng-xq/orgs",
            "received_events_url": "https://api.github.com/users/zheng-xq/received_events",
            "repos_url": "https://api.github.com/users/zheng-xq/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/zheng-xq/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/zheng-xq/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/zheng-xq"
        }
    ],
    "author_association": "NONE",
    "body": "Installing via:\n\n``` bash\n# For GPU-enabled version (only install this version if you have the CUDA sdk installed)\n$ pip install https://storage.googleapis.com/tensorflow/linux/gpu/tensorflow-0.5.0-cp27-none-linux_x86_64.whl\n```\n\nTried to run the alexnet_benchmark.py and it's looking for CUDA 7.0 specifically.\n\nI have CUDA 7.5 on my machine.\n\nFull stack:\n\n``` bash\nTraceback (most recent call last):\n  File \"alexnet_benchmark.py\", line 21, in <module>\n    import tensorflow.python.platform\n  File \"/home/awesomebox/anaconda/lib/python2.7/site-packages/tensorflow/__init__.py\", line 4, in <module>\n    from tensorflow.python import *\n  File \"/home/awesomebox/anaconda/lib/python2.7/site-packages/tensorflow/python/__init__.py\", line 22, in <module>\n    from tensorflow.python.client.client_lib import *\n  File \"/home/awesomebox/anaconda/lib/python2.7/site-packages/tensorflow/python/client/client_lib.py\", line 35, in <module>\n    from tensorflow.python.client.session import InteractiveSession\n  File \"/home/awesomebox/anaconda/lib/python2.7/site-packages/tensorflow/python/client/session.py\", line 11, in <module>\n    from tensorflow.python import pywrap_tensorflow as tf_session\n  File \"/home/awesomebox/anaconda/lib/python2.7/site-packages/tensorflow/python/pywrap_tensorflow.py\", line 28, in <module>\n    _pywrap_tensorflow = swig_import_helper()\n  File \"/home/awesomebox/anaconda/lib/python2.7/site-packages/tensorflow/python/pywrap_tensorflow.py\", line 24, in swig_import_helper\n    _mod = imp.load_module('_pywrap_tensorflow', fp, pathname, description)\nImportError: libcudart.so.7.0: cannot open shared object file: No such file or directory\n```\n\nTried the docker install, but the docker image is configured for a particular NVIDIA driver version, and doesn't work with others. (this is a known issue: docker driver version and system driver version must exactly match)\n",
    "closed_at": "2016-02-22T21:37:06Z",
    "closed_by": {
        "avatar_url": "https://avatars.githubusercontent.com/u/463737?v=4",
        "events_url": "https://api.github.com/users/vrv/events{/privacy}",
        "followers_url": "https://api.github.com/users/vrv/followers",
        "following_url": "https://api.github.com/users/vrv/following{/other_user}",
        "gists_url": "https://api.github.com/users/vrv/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/vrv",
        "id": 463737,
        "login": "vrv",
        "node_id": "MDQ6VXNlcjQ2MzczNw==",
        "organizations_url": "https://api.github.com/users/vrv/orgs",
        "received_events_url": "https://api.github.com/users/vrv/received_events",
        "repos_url": "https://api.github.com/users/vrv/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/vrv/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/vrv/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/vrv"
    },
    "comments": 54,
    "comments_url": "https://api.github.com/repos/tensorflow/tensorflow/issues/20/comments",
    "created_at": "2015-11-09T17:39:36Z",
    "events_url": "https://api.github.com/repos/tensorflow/tensorflow/issues/20/events",
    "html_url": "https://github.com/tensorflow/tensorflow/issues/20",
    "id": 115927762,
    "labels": [],
    "labels_url": "https://api.github.com/repos/tensorflow/tensorflow/issues/20/labels{/name}",
    "locked": false,
    "milestone": null,
    "node_id": "MDU6SXNzdWUxMTU5Mjc3NjI=",
    "number": 20,
    "performed_via_github_app": null,
    "reactions": {
        "+1": 0,
        "-1": 0,
        "confused": 0,
        "eyes": 0,
        "heart": 0,
        "hooray": 0,
        "laugh": 0,
        "rocket": 0,
        "total_count": 0,
        "url": "https://api.github.com/repos/tensorflow/tensorflow/issues/20/reactions"
    },
    "repository_url": "https://api.github.com/repos/tensorflow/tensorflow",
    "state": "closed",
    "state_reason": "completed",
    "timeline_url": "https://api.github.com/repos/tensorflow/tensorflow/issues/20/timeline",
    "title": "CUDA 7.5 fails with pip install and docker (Ubuntu 14.04)",
    "updated_at": "2016-10-03T21:26:06Z",
    "url": "https://api.github.com/repos/tensorflow/tensorflow/issues/20",
    "user": {
        "avatar_url": "https://avatars.githubusercontent.com/u/1310570?v=4",
        "events_url": "https://api.github.com/users/soumith/events{/privacy}",
        "followers_url": "https://api.github.com/users/soumith/followers",
        "following_url": "https://api.github.com/users/soumith/following{/other_user}",
        "gists_url": "https://api.github.com/users/soumith/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/soumith",
        "id": 1310570,
        "login": "soumith",
        "node_id": "MDQ6VXNlcjEzMTA1NzA=",
        "organizations_url": "https://api.github.com/users/soumith/orgs",
        "received_events_url": "https://api.github.com/users/soumith/received_events",
        "repos_url": "https://api.github.com/users/soumith/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/soumith/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/soumith/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/soumith"
    }
}